lib/output/html.js

```diff
@@ -57,30 +50,5 @@ function highlight(comment, options) {
 module.exports = function makeHTML(comments, options, callback) {
   options = options || {};
   comments = walk(comments, highlight, options);
-
-  var themeModule = resolveTheme(options.theme);
-  var pageTemplate = getTemplate(Handlebars, themeModule, 'index.hbs');
-
-  Handlebars.registerPartial('section',
-    getTemplate(Handlebars, themeModule, 'section.hbs'));
-
-  var paths = comments.map(function (comment) {
-    return comment.path.join('.');
-  }).filter(function (path) {
-    return path;
-  });
-
-  helpers(Handlebars, paths);
-
-  // push assets into the pipeline as well.
-  vfs.src([themeModule + '/assets/**'], { base: themeModule })
-    .pipe(concat(function (files) {
-      callback(null, files.concat(new File({
-        path: 'index.html',
-        contents: new Buffer(pageTemplate({
-          docs: comments,
-          options: options
-        }), 'utf8')
-      })));
-    }));
+  require(options.theme || 'documentation-theme-default')(comments, options, callback);
```

Comment:
Have you tested that this will work with relative paths with the CLI? I'm worried that specifying, for instance `-t ./my-theme` will look for that theme within `lib/output/`, rather than the CWD